scene/3d/navigation_region_3d.cpp

```diff
@@ -33,6 +33,12 @@
 #include "mesh_instance_3d.h"
 #include "servers/navigation_server_3d.h"
 
+#include "modules/modules_enabled.gen.h"
+
+#ifdef MODULE_NAVIGATION_MESH_GENERATOR_ENABLED
```

Comment:
If we've done the separation correctly, we shouldn't have to check if any module has been enabled here.

Checking for the presence of the default module defeats the purpose of allowing it to be disabled.

Reply:
Leftover from earlier iterations.